src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -268,7 +268,16 @@ bool SolverBase::execute( real64 const time_n,
     if( dtRemaining > 0.0 )
     {
       nextDt = setNextDt( dtAccepted, domain );
-      nextDt = std::min( nextDt, dtRemaining );
+      if( nextDt < dtRemaining )
+      {
+        // better to do two equal steps than one big and one small (even potentially tiny)
+        if( nextDt * 2 > dtRemaining )
```

Comment:
Shouldn't we be even stronger like 
```suggestion
        if( nextDt  > .7*dtRemaining )
```
?

Reply:
could you please clarify?

Reply:
Neverming you're spliting if `nextDt` is above .5*`dtRemaining` so you're already at most conservative threshold.